tests/unit-tests/util/class-wc-tests-core-functions.php

```diff
@@ -604,6 +604,28 @@ public function test_wc_get_template() {
 		$this->assertNotEmpty( $template );
 	}
 
+	/**
+	 * This test ensures that the absolute path to template files is replaced with a token. We do this so
+	 * that the path can be made relative to each installation, and the cache can be shared.
+	 */
+	public function test_wc_get_template_cleans_absolute_path() {
+		add_filter( 'woocommerce_locate_template', array( $this, 'force_template_path' ), 10, 2 );
+
+		ob_start();
+		try {
+			wc_get_template( 'global/wrapper-start.php' );
+		} catch ( \Exception $exception ) {
+			// Since the file doesn't really exist this is going to throw an exception (which is fine for our test).
+		}
+		ob_end_clean();
+
+		remove_filter( 'woocommerce_locatsdae_template', array( $this, 'force_template_path' ) );
```

Comment:
I think there's a small typo here in `woocommerce_locatsdae_template`? Should be `woocommerce_locate_template` yeah?

Reply:
Considering I copy/pasted the add_filter and renamed, I'm not entirely sure how that happened 😢 

Thanks for the catch!